app/views/health/self_sufficiency_matrix_forms/_show.haml

```diff
@@ -1,66 +1,64 @@
+          %dt.inline Case Worker:
+          %dd.inline= @form.user&.name
+      %dl
+        %dt.inline Point at which this assessment was completed:
+        - if @form.point_completed.present?
+          %dd.inline= @form.point_completed
+        - else
+          %dd.inline.font-italic.gray Not provided
+        %dd.breaker
+        - if @form.health_file.present?
+          %dt.inline File (see attached):
+          %dd.inline= link_to @form.health_file&.name, polymorphic_path([:download] + self_sufficiency_matrix_form_path_generator, client_id: @client.id, id: @form)
+    - if @form.editable_by?(current_user)
+      .ml-auto
+        = link_to polymorphic_path([:edit] + self_sufficiency_matrix_form_path_generator, id: @form.id), class: 'btn btn-secondary btn-sm', disabled: @claim_submitted, data: { toggle: 'tooltip' }, title: (_('A claim was submitted for this SSM; it is no longer editable.') if @claim_submitted) do
+          %i.icon-pencil
+          Edit SSM
 
-  %dl
-    %dt.inline Name:
-    %dd.inline= @client.name
-    %dd.breaker
-    %dt.inline Date Completed:
-    %dd.inline= @form.completed_at&.to_date || 'in progress'
-    %dd.breaker
-    %dt.inline Case Worker:
-    %dd.inline= @form.user&.name
-    %dd.breaker
-    - if @form.health_file.present?
-      %dt.inline File (see attached):
-      %dd.inline= link_to @form.health_file&.name, polymorphic_path([:download] + self_sufficiency_matrix_form_path_generator, client_id: @client.id, id: @form)
-
-  %p.text-right(style="font-size:smaller;font-style:italic;")
+  %p.text-right.gray(style="font-size:smaller;font-style:italic;")
     Only answered questions are shown.
 
-
-
-  - if @form.point_completed.present?
-    .client__assessment-overall-summary
-      .client__assessment-question
-        Point at which this assessment was completed
-      .client__assessment-answer
-        = @form.point_completed
-
   - Health::SelfSufficiencyMatrixForm::SECTIONS.keys.each do |section_key|
     - score = @form.send "#{section_key}_score"
     - if score.present?
-      .client__assessment-section-summary
-        - title = Health::SelfSufficiencyMatrixForm::SECTION_TITLES[section_key]
-        .client__assessment-question.client__assessment-question--title
-          = title
-        %dl.client__assessment-question-summary
-          %dt.client__assessment-question.client__assessment-question--score.inline
-            Score
-          %dd.client__assessment-question.client__assessment-question--score.inline
-            = score
-          %dd.client__assessment-answer.client__assessment-answer--text
-            = Health::SelfSufficiencyMatrixForm::SECTIONS[section_key][score]
-          %dt.client__assessment-question.client__assessment-answer--notes
-            = "Notes"
-          %dd.client__assessment-answer.client__assessment-answer--notes
-            = simple_format @form.send("#{section_key}_notes")
-
-  %hr
+      .ssm__row
+        %div{style: 'display: flex;'}
```

Comment:
is this different from:
```suggestion
        .d-flex
```

Reply:
We don't have `.d-flex` class defined in the `print.scss` file. But I can add it, so we can use it.

Reply:
seems like a nice thing to have since it's tiny and we have it in the rest of the app.